build/lin.sh

```diff
@@ -293,17 +339,54 @@ cd ${DEPS}/vips
 ./configure --host=${CHOST} --prefix=${TARGET} --enable-shared --disable-static --disable-dependency-tracking \
   --disable-debug --disable-introspection --without-analyze --without-cfitsio --without-fftw --without-heif \
   --without-imagequant --without-magick --without-matio --without-nifti --without-OpenEXR --without-openslide \
-  --without-pdfium --without-poppler --without-ppm --without-radiance \
-  --with-zip-includes=${TARGET}/include --with-zip-libraries=${TARGET}/lib \
-  --with-jpeg-includes=${TARGET}/include --with-jpeg-libraries=${TARGET}/lib
+  --without-pdfium --without-poppler --without-ppm --without-radiance
+# https://docs.fedoraproject.org/en-US/packaging-guidelines/#_removing_rpath
+sed -i'.bak' 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' libtool
 make install-strip
 
 # Cleanup
 rm -rf ${TARGET}/lib/{pkgconfig,.libs,*.la,cmake}
-cd ${TARGET}/include
 
-# Set RPATH to $ORIGIN
-find ${TARGET}/lib -type f -name "*.so*" -exec sh -c "patchelf --set-rpath '\$ORIGIN' --force-rpath {}" \;
+mkdir ${TARGET}/lib-filterd
```

Comment:
Nitpick: `filterd` sounds like a subgenre of French house music - did you mean `filtered`?

Reply:
Ah yes, a silly typo. Fixed with commit https://github.com/lovell/sharp-libvips/pull/44/commits/3dbab0f89b50f3126bc43928663e782847fdfae8.